src/main/kotlin/io/zerobase/smarttracing/resources/OrganizationsResource.kt

```diff
@@ -74,11 +75,10 @@ class OrganizationsResource(val dao: GraphDao, private val siteTypes: MultiMap<S
                 hasMultipleSites
             )
             return id?.let { IdWrapper(id) }
-        } catch (e: Exception) {
-            val res = Response.status(Response.Status.BAD_REQUEST)
-                .entity(e.message)
-                .build()
-            throw WebApplicationException(res)
+        } catch (e: InvalidPhoneNumberException) {
+            throw BadRequestException(e.message)
+        } finally {
+            throw BadRequestException("Unknown exception")
```

Comment:
this will throw an exception a the method is trying to return a valid value. that's not what we want

Reply:
i thought that it will return, and not go into finally, but either way fixed that